guide/phone_store/phone_store_service.bal

```diff
@@ -0,0 +1,241 @@
+import ballerina/jms;
+
+//Deploying on kubernetes
+
+//import ballerinax/kubernetes;
+// Other imports
+
+// Type definition for a phone order
+
+//json[] phoneInventory = ["Apple:190000", "Samsung:150000", "Nokia:80000", "HTC:40000", "Huawei:100000"];
+
+// 'jms:Connection' definition
+
+// 'jms:Session' definition
+
+// 'jms:QueueSender' endpoint definition
+
+//@kubernetes:Ingress {
+//hostname:"ballerina.guides.io",
+//name:"ballerina-guides-phone_store_service",
+//path:"/"
+//}
+
+//@kubernetes:Service {
+//serviceType:"NodePort",
+//name:"ballerina-guides-phone_store_service"
+//}
+
+//@kubernetes:Deployment {
+//image:"ballerina.guides.io/phone_store_service:v1.0",
+//name:"ballerina-guides-phone_store_service"
+//}
+
+//endpoint http:Listener listener {
+//port:9090
+//};
+
+//@http:ServiceConfig {basePath:"/phonestore"}
+//service<http:Service> phone_store_service bind listener {
+
+//Deploying on docker
+
+// import ballerinax/docker;
+// Other imports
+
+// Type definition for a phone order
+
+//json[] phoneInventory = ["Apple:190000", "Samsung:150000", "Nokia:80000", "HTC:40000", "Huawei:100000"];
+
+// 'jms:Connection' definition
+
+// 'jms:Session' definition
+
+// 'jms:QueueSender' endpoint definition
+
+//@docker:Config {
+//registry:"ballerina.guides.io",
+//name:"phone_store_service",
+//tag:"v1.0"
+//}
+
+// Service endpoint
+//@docker:Expose{}
+//endpoint http:Listener listener {
+//port:9090
+//};
+
+// phone store service, which allows users to order phones online for delivery
+//@http:ServiceConfig {basePath:"/phonestore"}
+//service<http:Service> phone_store_service bind listener {
+
+public http:Request backendreq;
+
+// Type definition for a phone order
+type phoneOrder record {
+    string customerName;
+    string address;
+    string contactNumber;
+    string orderedPhoneName;
+};
+
+// Global variable containing all the available phones
+json[] phoneInventory = ["Apple:190000", "Samsung:150000", "Nokia:80000", "HTC:40000", "Huawei:100000"];
+
+// Initialize a JMS connection with the provider
+// 'providerUrl' and 'initialContextFactory' vary based on the JMS provider you use
+// 'Apache ActiveMQ' has been used as the message broker in this example
+jms:Connection jmsConnection = new({
+        initialContextFactory: "org.apache.activemq.jndi.ActiveMQInitialContextFactory",
+        providerUrl: "tcp://localhost:61616"
+    });
+
+// Initialize a JMS session on top of the created connection
+jms:Session jmsSession = new(jmsConnection, {
+        acknowledgementMode: "AUTO_ACKNOWLEDGE"
+    });
+
+// Initialize a queue sender using the created session
+endpoint jms:QueueSender jmsProducer {
+    session:jmsSession,
+    queueName:"OrderQueue"
+};
+
+// Service endpoint
+endpoint http:Listener listener {
+    port:9090
+};
+
+// phone store service, which allows users to order phones online for delivery
+@http:ServiceConfig {basePath:"/phonestore"}
+service<http:Service> phone_store_service bind listener {
```

Comment:
phoneStoreService is better than using underscore

Reply:
done